scvi/lightning/_trainingplans.py

```diff
@@ -69,6 +71,7 @@ def __init__(
         lr_scheduler_metric: Literal[
             "elbo_validation", "reconstruction_loss_validation", "kl_local_validation"
         ] = "elbo_validation",
+        lr_min: float = None,
```

Comment:
default should be 0 like in pytorch right?

Reply:
whoops ya